components/raftstore-v2/src/raft/apply.rs

```diff
@@ -59,6 +62,8 @@ pub struct Apply<EK: KvEngine, R> {
     res_reporter: R,
     read_scheduler: Scheduler<ReadTask<EK>>,
     sst_importer: Arc<SstImporter>,
+    observe_info: CmdObserveInfo,
```

Comment:
(I'm also not pretty sure about how this influences to the apply. Will the logic of observing be sent in another PR?)

Reply:
Yes, see https://github.com/tikv/tikv/pull/14562